CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for escaping in attribiute values in LabelMe format (<https://github.com/openvinotoolkit/datumaro/issues/49>)
 - Support for Segmentation Splitting (<https://github.com/openvinotoolkit/datumaro/pull/223>)
 - Support for CIFAR-10/100 dataset format (<https://github.com/openvinotoolkit/datumaro/pull/225>)
+- Support import COCO panoptic and stuff format (<https://github.com/openvinotoolkit/datumaro/pull/210>, <https://github.com/openvinotoolkit/cvat/issues/1784>, <https://github.com/openvinotoolkit/cvat/issues/1143>)
```

Comment:
Adding support in Datumaro wont automatically add it in CVAT. It will need at least updating a Datumaro revision in CVAT, which is fixed. It will allow importing, but I'd expect a new format in CVAT for coco segmentation.

Reply:
you're right. fixed